regulations/templates/regulations/toc-section.html

```diff
@@ -1 +1,3 @@
-<li><a href="{{item.url}}" data-section-id="{{item.section_id}}" id="nav-{{item.section_id}}"><span class="toc-section-marker">§&nbsp;{{item.label}}</span> {{item.sub_label|safe}}</a></li>
+<li><a href="{{item.url}}" data-section-id="{{item.section_id}}" id="nav-{{item.section_id}}">
+        <span class="toc-section-marker">{% if item.is_section_span%}§§&nbsp;{% else %}§&nbsp;{% endif %} {{item.label}}</span> {{item.sub_label|safe}}
```

Comment:
I think we'll need to remove the space after the `endif` here -- the goal is to not allow the browser to break `§ 1234.56` onto two lines.